TrackController/src/trackCtrl/view/Controller.java

```diff
@@ -1,5 +1,6 @@
 package trackctrl.view;
```

Comment:
package is listed as trackctrl.view and package name is trackCtrl in the path. Is it big C or little c? lol